### PR TITLE
Test for Promo Module on Desktop and Mobile

### PR DESCRIPTION
**Pull request by AndreiH**
This is for Issue https://github.com/mozilla/marketplace-tests/issues/604

### CODE REVIEW

File: tests/mobile/test_home_page.py [hunk 1]
```diff
@@ -13,6 +13,15 @@
 class TestHomepage():
 
     @pytest.mark.nondestructive
+    def test_that_promo_module_not_present_on_mobile(self, mozwebqa):
+
+        home_page = Home(mozwebqa)
+        home_page.go_to_homepage()
+
+        Assert.true(home_page.is_promo_box_not_visible)
+        Assert.false(home_page.is_promo_box_present)
```
Comment (bobsilverberg) [inline tests/mobile/test_home_page.py]: When run with Saucelabs, the test fails here, with `home_page.is_promo_box_present` returning `True`. Do we know for sure that the element will not be present, or is it enough to check that it's not visible?

Stack trace:
```tests/mobile/test_home_page.py F

================================================================= FAILURES ==================================================================
_________________________________________ TestHomepage.test_that_promo_module_not_present_on_mobile _________________________________________

self = <tests.mobile.test_home_page.TestHomepage instance at 0x105cbeb48>
mozwebqa = <pytest_mozwebqa.pytest_mozwebqa.TestSetup instance at 0x105cbecf8>

    @pytest.mark.nondestructive
    def test_that_promo_module_not_present_on_mobile(self, mozwebqa):

        home_page = Home(mozwebqa)
        home_page.go_to_homepage()

        Assert.true(home_page.is_promo_box_not_visible)
>       Assert.false(home_page.is_promo_box_present)

tests/mobile/test_home_page.py:22:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <class 'unittestzero.Assert'>, first = True, msg = None

    @classmethod
    def false(self, first, msg=None):
        """
            Asserts that what is given is equal to False

            :Args:
             - First object to be tested
             - Message that will be printed if it fails
            """

>       assert bool(first) is False, msg
E       AssertionError: None

../../.virtualenvs/marketplace-tests/lib/python2.7/site-packages/unittestzero.py:91: AssertionError
```

**Comment by bobsilverberg** [comment]
@AndreiH This looks good and `test_that_promo_module_is_visible` passes fine for me on all 3 environments, but `test_that_promo_module_not_present_on_mobile` is failing for me. I added a comment to the diff.

**Comment by AndreiH** [comment]
Strange because I couldn't find the element in the HTML on Mobile.
But it failed for me also. I removed that assert @bobsilverberg . Thanks!

**Comment by viorelaioia** [comment]
lgtm, merging!

**Pull request merged**